Code/Engine/Foundation/Math/Implementation/Frustum.cpp

```diff
@@ -258,6 +259,7 @@ ezFrustum ezFrustum::MakeFromMVP(const ezMat4& mModelViewProjection0, ezClipSpac
   ezFrustum frustum;
   const ezResult res = TryMakeFromMVP(frustum, mModelViewProjection0, depthRange, handedness);
   EZ_ASSERT_DEV(res.Succeeded() && frustum.IsValid(), "Frustum is not valid after construction.");
+  (void)res;
   return frustum;
```

Comment:
Remove. Also all others.

Reply:
Without this shipping builds will no longer compile as `res` becomes an unused variable. As an alternative I could use `EZ_VERIFY` not sure if that conveys the same intention?

Reply:
EZ_IGNORE_UNUSED

Reply:
Thanks, fixed!